### PR TITLE
Add experimental spread setup for trusty.

### PR DESCRIPTION
**Pull request by vosst**

### CODE REVIEW

File: spread.yaml [hunk 1]
```diff
@@ -22,6 +22,18 @@ prepare: |
     echo "Spread is running as $(id)"
     [ "$REUSE_PROJECT" != 1 ] || exit 0
     ./spread-tests/spread-prepare.sh
+    # trusty support is under development right now
+    # we special-case the release until we have officially landed
+    if ["$(lsb_release -sc)" == "trusty"]
```
Comment (zyga) [inline spread.yaml]: `lsb_release` is not preinstalled everywhere, please move this to `spread-prepare.sh` where we already have all the information `release_ID` and `release_VERSION_ID`. Please do that before we `apt-get install snapd`.

File: spread.yaml [hunk 2]
```diff
@@ -22,6 +22,18 @@ prepare: |
...
+    then
+        add-apt-repository ppa:thomas-voss/trusty
+        apt update && apt upgrade
+        apt install systemd snapd
```
Comment (zyga) [inline spread.yaml]: systemd should be a dependency of snapd but I'm okay if that's just a temporary thing.

File: spread.yaml [hunk 3]
```diff
@@ -22,6 +22,18 @@ prepare: |
...
+        # he following two lines are temporary and meant to go away
+        # once we have finalized trusty support.
+        service systemd start
+        apparmor_parser -R /etc/apparmor.d/usr.lib.snapd.snap-confine
```
Comment (zyga) [inline spread.yaml]: Why do you need this here? Trusty support should be good in the tree now .

Comment (vosst) [inline spread.yaml]: c'n'p leftover, good catch.

File: spread.yaml (hunk 2 above)
Comment (vosst) [inline spread.yaml]: Yup, working on getting the dependency in place correctly.

**Pull request merged**